frontend/src/pages/spotnet/overview/Overview.jsx

```diff
@@ -0,0 +1,120 @@
+import React from "react";
+import ArrowDownDouble from "../../../assets/icons/arrow-down-double.svg"
+import "./overview.css";
+import { Button, Image } from "react-bootstrap";
+import { Link, useLocation } from 'react-router-dom';
+
+
+const OverviewPage = () => {
+    const location = useLocation();
+    const activeHash = location.hash
+
+    console.log(activeHash)
+
+    const handleScrollDown = () => {
+        const scrollAmount = document.documentElement.scrollHeight * 0.3;
+
+        window.scrollBy({
+            top: scrollAmount,
+            behavior: "smooth"
+        });
+    }
+
+    return (
+        <div className="overview-container">
+            <nav className="sidebar">
+                <h2 className="sidebar-title">Content</h2>
+                <div>
+
+                    <ul className="sidebar-list">
```

Comment:
it should use TableOfContents, just like documentation page

Reply:
I'm now using the TableOfContents component in the overview page, but it seems it is not totally reusable as some default paths were passed in the component. Should I still proceed?

Reply:
what do you mean exactly @jahrulezfrancis point on that is blocking you

Reply:
```
 useEffect(() => {
    if (location.pathname === '/documentation') {
      setActiveId('introduction');
    }

    const handleScroll = () => {
      const sections = document.querySelectorAll('.documentation-section');
      const scrollPosition = window.scrollY;

      sections.forEach((section) => {
        const sectionTop = section.offsetTop - 100;
        const sectionHeight = section.offsetHeight;

        if (scrollPosition >= sectionTop && scrollPosition < sectionTop + sectionHeight) {
          setActiveId(section.id);
        }
      });
    };

```

The above useEffect is in the TableOfContent comoponent and the path won't work efficiently in Overview page.

Reply:
so just pass everything that is generic as a prop from both Overview/Documentation pages to TableOfContent and use here

Reply:
Noted. I'll be on it